test/e2e/_helpers.js

```diff
@@ -17,14 +18,19 @@ export const scrollToTop = ClientFunction(() => {
   window.scrollTo(0, 0)
 })
 
+const filterDisabled = referenceDataHelpers.filterDisabled()
+
 export async function generatePerson(overrides = {}) {
   const firstNames = faker.name.firstName()
   const lastName = faker.name.lastName()
 
   let genders = await referenceDataService.getGenders()
-  // TODO: implement proper test to render this filter unnecessary
-  genders = genders.filter(gender => gender.title.match(/^(Male|Female)$/i))
-  const ethnicities = await referenceDataService.getEthnicities()
+  // TODO: implement proper test to render male/female filter unnecessary
+  genders = genders
+    .filter(filterDisabled)
```

Comment:
I think it would be good to also add this filter to all other calls to reference data to prevent this happening  with other datasets. 

From a quick scan it looks like we make a similar call in [`createPersonFixture`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/3c5538c7c9d8a66c90d3e2fbba93c5263e24b614/test/e2e/_helpers.js#L66-L67).

So might be worth having a search for calls to the `referenceDataService` as I'm sure we're doing calls to locations, etc too?

We designed all the reference data endpoints in the API to return the same format and include a `disabled_at` attribute so that they could be consistently filtered out in the way to support us changing the data over time.

Reply:
For the call in createPersonFIxture you reference, we then immediately filter it down by the value that's been passed in so it wouldn't be an issue there, but the getLocations call could indeed be susceptible to the same issue, so I've added filtered versions of the methods in the _helpers file to call rather than the refData ones directly.

I have a niggling feeling this should be the default behaviour of the refData methods, and that we should ask for the unfiltered list explicitly...

Reply:
I was going to suggest the same after I saw your changes 😂 

Now that we're calling it in multiple places it should probably be something rolled into the service rather than where the service is called to prevent unfiltered results being returned again in the future.

Up to you if you want to do as part of this or add a GH issue/TODO to track it.